src/rules/apex/apex-import.ts

```diff
@@ -23,12 +24,13 @@ export const rule = ESLintUtils.RuleCreator.withoutDocs({
     },
     meta: {
         docs: {
+            url: getDocUrl(APEX_IMPORT_RULE_ID),
             description:
                 'Using Apex in LWC Offline-enabled mobile apps requires additional considerations to ensure proper functioning in offline scenarios. See Use Apex While Mobile and Offline (https://developer.salesforce.com/docs/atlas.en-us.mobile_offline.meta/mobile_offline/apex.htm) for more details. GraphQL API is a new paradigm of sending and receiving data. This API allows developers to interact with the Salesforce Platform through GraphQL, a standard query language for APIs and a runtime for fulfilling those queries with your data. Learn more about creating SOQL queries with GraphQL that may work better for mobile offline use cases. https://developer.salesforce.com/docs/platform/graphql/guide/graphql-wire-lwc.html'
```

Comment:
Hi @khawkins,  need your help on the message or description wording polish. thx

Reply:
It feels like in a world where we have backing "long-form" docs for each rule, saying less in the inline messaging is probably better, deferring the more detailed content to the long-form doc.

Reply:
So yeah, both `meta > docs > description` (here), and `meta > messages > [APEX_IMPORT_RULE_ID]` (below) are too long. These should effectively be one-liners. Consider ESLint's `array-callback-return` docs [configuration](https://github.com/eslint/eslint/blob/6f37b0747a14dfa9a9e3bdebc5caed1f39b6b0e2/lib/rules/array-callback-return.js#L224-L228) as an example. Their `description` value shows up in their extended docs here:

![image](https://github.com/user-attachments/assets/209c1505-5f21-49fa-9675-eebdeeba2f04)

Whereas clicking into the configured `url` link leads to a [detailed doc page](https://eslint.org/docs/v8.x/rules/array-callback-return). We should be shooting for a similar approach, leaving the `description` value as something like:

> Using Apex in LWC Offline-enabled mobile apps requires additional considerations to ensure proper functioning in offline scenarios.

And then using the backing doc to explain the rest of the details.

Reply:
thx @khawkins make sense to shorten the msg to short one line and link to a fat/meaty external doc.  

@clivewong you are writing the meaty doc. Mind to give a one line for the written or to be written doc, thx